srcpkgs/linux6.12/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'linux6.12'
 pkgname=linux6.12
 version=6.12.6
-revision=1
+revision=2
```

Comment:
needs rebase: https://github.com/void-linux/void-packages/commit/46e06acc14117c0c676912a4b728e6331eeb856b

Reply:
didn't realize was out of date my bad